src/query.js

```diff
@@ -1032,8 +1032,7 @@ function CreateRole(params) {
  *     - name: A valid schema name
  *     - issuer: A unique string
  *     - jwks_uri: A valid HTTPS URI
- *     - allowed_roles: An optional list of Role refs
- *     - allowed_collections: An optional list of user-defined Collection refs
+ *     - membership: An array of roles
```

Comment:
Array of roles doesn't convey the correct purpose of this field. It should say it accepts an object with a pair of role/predicate and that the predicate returns true or false.